clustered/client/src/test/java/org/ehcache/clustered/client/internal/store/ServerStoreProxyTest.java

```diff
@@ -160,6 +160,19 @@ public void testReplaceAtHeadSuccessFull() {
     assertChainHas(anotherReplace, 800L, 4000L, 40000L);
   }
 
+  @Test
+  public void testClear() {
+    serverStoreProxy.append(1L, createPayload(100L));
+
+    Chain chain = serverStoreProxy.get(1);
+    assertThat(chain.isEmpty(), is(false));
```

Comment:
These assert are not needed here, the fact that append works should be tested elsewhere already.